playbooks/domon-https.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create objects in Icinga
+  hosts: localhost
+  gather_facts: false
+  vars:
+    object_name: domon-https
+  tasks:
+    # icinga_service_template
+    - name: "Icinga_service_template-{{ object_name }}"
+      ansible.builtin.include_role:
+        name: "telekom_mms.icinga_director.ansible_icinga"
+        tasks_from: "icinga_service_template"
+      vars:
+        icinga_service_templates:
+          - service_template_object:
+              - "{{ object_name }}"
+            imports:
+              - domon-non-agent-service
```

Comment:
This object is only available on our instances. Maybe we should use a variable for this too.

Reply:
All required service templates are included in the playbooks, now